frontend/src/pages/GraphicalSelector/utils.js

```diff
@@ -0,0 +1,13 @@
+import GRAPH_STYLE from "./config";
+
+// plannedCourses is an object of with keys of courseCodes
+const handleNodeData = (courseCode, plannedCourses) => {
+  // determine if planned or unplanned
+  if (Object.prototype.hasOwnProperty.call(plannedCourses, courseCode)) {
```

Comment:
Nitpick: why dont u do `plannedCourses[courseCode]` instead? 👀  Either way works just curios lol

Reply:
truee lul - tbh I used hasOwnProperty before but the linter didn't like it so I googled and they recommended this xd